compose.yml

```
@@ -0,0 +1,17 @@
+services:
+  php:
+    image: ghcr.io/sylius/sylius-php:8.2-alpine
+  mysql:
+    image: mysql:5.7 # Sylius is fully working on mysql 8.0 version
```

```suggestion
    image: mysql:8.0 # Sylius is fully working on mysql 8.0 version
```

AFAIR we're bumping the minimal mysql version in 1.13, previous versions are working with `8.0` so we can safely use it.